### DISCUSSION TITLE
Sort issues by state

### DISCUSSION THREAD

**Discussion by SergioGracia** [rozpuszczalny/vscode-redmine#48] [Ideas]
Hello! I just discovered this extension and I think it's fantastic. I am an avid user of Redmine and this comes very very handy in my daily rutine.

I would like to ask: is there a possibility of adding an ordering of issues by status? In my case I have many open tickets that are intermingled and it would be great if I could sort them in the extension panel.

Greetings and thanks!

**Comment by rozpuszczalny** [comment]
Hey, thank you for the kind words!

I think this would be great to have a possibility to sort issues in the panel as well as in the quick pick menu.

While it's rather simple to place a sort button in the panel with different possibilities to sort by, I wonder how it could be done in the quick pick menu. Should it be a separate command like "Redmine: change sorting for quick pick menu", an option in settings, or reuse the sorting from the panel? Do you have a preference or you don't care, since your workflow is based mainly on the panel?

**Reply by SergioGracia** [reply]
Hello again!
In my specific case I would like to be able to order from the panel. The quickpick menu is rarely used other than to go directly to a file or change the theme of the editor but I understand that there are people who are interested in being able to do it on both sides.

Personally, I would be delighted to be able to order in the extension's own panel and to be able to do it by status or priority.

Greetings!
